wdl-format/tests/format/ENCODE-DCC_chip-seq-pipeline/source.formatted.wdl

```diff
@@ -1841,13 +1659,10 @@ workflow chip {
                 runtime_environment = runtime_environment,
             }
         }
-        File? ctl_bam_ = (
-            if has_output_of_align_ctl
-            then ctl_bams[i]
-            else align_ctl.bam
-        )
+        File? ctl_bam_ = if has_output_of_align_ctl then ctl_bams[i] else align_ctl.bam
 
-        Boolean has_input_of_filter_ctl = has_output_of_align_ctl || defined(align_ctl.bam)
+        Boolean has_input_of_filter_ctl = has_output_of_align_ctl || defined(align_ctl.bam
+            )
```

Comment:
And here. Sorry for so many comments, just want to call out places that require further examination.